web_src/css/helpers.css

```diff
@@ -137,6 +138,7 @@ Gitea's private styles use `g-` prefix.
 .gt-m-4 { margin: 1rem !important; }
 .gt-m-5 { margin: 2rem !important; }
 
+.gt-ml--2 { margin-left: -.25rem !important; }
```

Comment:
Maybe it would be good to discuss such things and write in the docu which styles are allowed and when use helpers
In every PR I get a comment like 'use helper for this' and 'dont use helper for this'

Reply:
Negative margin is something to generally avoid yes, but in rare cass it's justified.

Please convert this class to tailwind, I think it'll be `-tw-ml-1`.